build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python3 starfyre --dev=True --path="test-application/" && python -m starfyre --build=True --path="test-application/"
+python3 starfyre --build --path="test-application"
```

Comment:
One suggestion - we should remove the `dev` flag from the build script once we remove it from the `main()` function.